examples/nextjs/src/utils/gql.js

```diff
@@ -79,22 +69,7 @@ const getGraphQLPageQuery = ({ path, language_id, mode}) => {
                     }
                 }
             }
-            template {
-                iDate
-                inode
-                identifier
-                source
-                title
-                friendlyName
-                modDate
-                sortOrder
-                showOnMenu
-                image
-                drawed
-                drawedBody
-            }
```

Comment:
We are not using the `template` in the `Angular` and `NextJS` libraries. It should be a safe change **but** we might want to check that we are not getting any `TypeScript` errors.